indico/modules/events/registration/templates/display/conference_home.html

```diff
@@ -23,6 +23,15 @@
                             </a>
                         </li>
                     {% endif %}
+                    {% if regform.is_apple_wallet_available %}
+                        <li>
+                            <a href="{{ url_for('event_registration.ticket_apple_wallet', regform) }}"
+                               class="icon-ticket"
+                               title="{% trans title=regform.title -%}Download ticket for the '{{ title }}' registration form for Apple Pass{%- endtrans %}">
+                                {% trans %}Apple Pass{% endtrans %} - {{ regform.title }}
```

Comment:
What do you think about mentioning "Passbook" here in addition to Apple Pass/Wallet?

I think this may be useful for people who do not actually use an apple device, but have an app that can show Passbook files (many Android users have one, even though ofc Google Wallet makes more sense for those who have it).

Reply:
Done.